Docs fix (#175)
tradingview/lightweight-charts: docs/customization.md

});
```

## Crosshair

> Crosshair shows an intersection of a price and time axis values on any hovered point on the chart.

Reviewer: ```suggestion
The crosshair shows the intersection of the price and time scale values at any point on the chart.
```
